src/tracer.ts

```diff
@@ -130,7 +151,10 @@ export default class Tracer {
      * @param  {any} carrier - see the documentation for the chosen `format`
      *         for a description of the carrier object.
      */
-    inject(spanContext, format, carrier) {
+    inject(spanContext: SpanContext | Span, format: typeof Constants.FORMAT_TEXT_MAP, carrier: { [key: string]: string }): void;
+    inject(spanContext: SpanContext | Span, format: typeof Constants.FORMAT_BINARY, carrier: { buffer: number[] }): void;
```

Comment:
I wasn't sure because the documentation doesn't specify this. It just says an "Array-like" that carries binary data. I interpreted that as a byte array, eg https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array or NodeJS Buffer (which would be compatible with `number[]` / `Array<number>`)